doc/qbk/02_2_strings.qbk

```diff
@@ -12,22 +13,58 @@
 [section:strings Using Strings]
 
 Modifiable sequences of characters are represented using instances of type
-[link json.ref.boost__json__string `string`].
-These strings have an interface almost identical to `std::string`, and
-operate the same way, with these variations:
+__string__.
+These strings have an modified `string_view` based version of the `std::string` interface, and
+operate the same way, except that:
 
-* __string__ is not a class template.
+* __string__ is not a class template,
+
+* __string__ only uses `char` as its character type,
+
+* redundant overloads for string operations have been replaced with a `string_view` based interface,
+
+* access to characters in the range `[size(), capacity()]` is permitted, and
 
 * __storage__ is used instead of __Allocator__.
 
-* New characters are 
+With the `string_view` based interface, operations requiring an input string are implemented as a single overload with
+a parameter of type `string_view`, and can accept most string-like objects in addition to other `string_view`s. 
+Objects such as null terminated character pointers, `std::string`, `json::string`, subranges of strings, and objects
+convertible to `string_view` can all be passed to these functions. 
 
-A string may be constructed initially empty without incurrening any
-memory allocations using the default storage, or a specified storage:
+[snippet_strings_4]
 
-[snippet_strings_1]
+More formally, `std::string` member function overloads that accept any of the
+following parameter combinations as an input string:
+
+* a `std::string` parameter, or
+
+* a `std::string` parameter and two `size_type` parameters that specify a substring within that string, or
```

Comment:
"string" three times in a sentence

Reply:
That's the formal spec of what actually happens, so it has to be redundant.

Reply:
Here is my quick try for the whole section

```
More formally, member function overloads for `std::string` accept either of the following parameter sets:

* object of `std::string` type

* object of `std::string` type and two `size_type` parameters that specify position of the first character and length of the substring in the object

* object of type convertible to __string_view__

* object of type convertible to __string_view__ and two `size_type` parameters that specify position of the first character and length of the substring in the object

* `const_pointer` to null-terminated character string
```